apps/ledger-live-mobile/src/live-common-setup.ts

```diff
@@ -101,13 +103,16 @@ registerTransportModule({
     id.startsWith("usb|")
       ? Promise.resolve() // nothing to do
       : null,
-  discovery: Observable.create(o => HIDTransport.listen(o)).pipe(
+  discovery: new Observable<DescriptorEvent<any>>(o =>
```

Comment:
Please do not use `any`.

Reply:
using `unknown` as this type is not very revelant for the context here